content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -12,40 +12,98 @@ Workspaces represent running infrastructure managed by Terraform.
 
 ## Create a Workspace
 
-This endpoint is used to create a new workspace either with or without a VCS configuration.
+`POST /organizations/:organization_name/workspaces`
 
-| Method | Path           |
-| :----- | :------------- |
-| POST | /organizations/:organization_name/workspaces |
+| Parameter            | Description                                                                                                                                                              |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `:organization_name` | The name of the organization to create the something in. The organization must already exist in the system, and the user must have permissions to create new somethings. |
 
-### Parameters
+### Request Body
 
-- `:organization_name` (`string: <required>`) - Specifies the organization name under which to create the workspace. The organization must already exist in the system, and the user must have permissions to create new workspaces. This parameter is specified in the URL path.
-- `name`
+This POST endpoint requires a JSON object with the following properties as a request payload.
 
-### Create a Workspace Without a VCS Repository
+Properties without a default value are required.
 
-If you supply nothing but a name, you can create a workspace without configuring it against a VCS repository.
+By supplying the necessary attributes under a `vcs-repository` object, you can create a Workspace that is configured against a VCS Repository.
```

Comment:
I've been pushing to NOT capitalize API constructs as proper nouns. That might be a losing fight tho, since everyone who codes the API seems to spontaneously capitalize them like "Workspace." (BTW, why is that? Is it because they're Ruby class names under the hood? Did everyone take German in college?)

Reply:
Yeah, I can make that change. This was the original sentence, so happy to bring it towards non-capitalisation.